playground/src/App.tsx

```diff
@@ -7,19 +7,19 @@ import ExplainerPage from "./pages/Explainer";
 function App() {
   return (
     <>
-      <Router>
+      <HashRouter>
         <NavBar />
         <Routes>
           <Route path="/examples/" element={<CodingEnvironment />} />
```

Comment:
I have `path="/examples"` with the `/` at the end. Not sure if this is important?